spec/order_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/order.rb'
+
+describe Order do
+  let(:basket) { { :Burger => 2 } }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)